packages/shared/src/components/modals/NewSourceModal.tsx

```diff
@@ -262,26 +258,41 @@ export default function NewSourceModal(props: ModalProps): ReactElement {
           </>
         )}
       </Modal.Body>
-      {(!!feeds?.length || isScraping) && (
-        <Modal.Footer>
-          {isScraping && (
-            <div className="flex justify-center w-full">
-              <Loader invertColor />
-            </div>
-          )}
-          {!!feeds?.length && (
+      <Modal.Footer>
```

Comment:
Should this not have justify between for the one with two btutons? (I do see you used mr-auto)

Reply:
Dont they have the same outcome?